runtime/mainnet/src/pos/currency.rs

```diff
@@ -26,8 +26,13 @@ parameter_types! {
 	pub(crate) const DefaultTreasuryCommissionFromTips: Perbill = Perbill::from_percent(25);
 }
 
-// 1 julian year to address leap years
-const YEAR_IN_MILLIS: u64 = 1000 * 3600 * 24 * 36525 / 100;
+pub trait CurrencyInfo {
```

Comment:
For reviewers:
I wanted to decouple pallets. Also, it makes it more clean in my understanding.